### DISCUSSION TITLE
Add Saturation preview when holding food

### DISCUSSION THREAD

**Discussion by StefanVonRanda** [Zrips/CMI#520] [Feature Requests]
When a player holds food, it shows you the amount of saturation a food item regenerates. like showed below.

![a3b042e4745985239e873b4fe6019655765ab69b](https://user-images.githubusercontent.com/16295563/41534115-a3021f40-72fd-11e8-9554-336e6f2c8e4a.gif)

Plugin the concept is based on.
https://www.spigotmc.org/resources/saturationpreview.21779/

**Comment by CloudeLecaw** [comment]
What a cool idea :D
